jovian/__init__.py

```diff
@@ -23,7 +23,40 @@
 
 
 def reset():
-    """Reset the tracked hyperparameters & metrics (for a fresh experiment)"""
+    """Reset the tracked hyperparameters & metrics (for a fresh experiment)
+    Example
```

Comment:
Maybe you should add Example in the next after a line break. It hasn't picked up example as Section header.

![Screenshot from 2019-09-25 14-46-23](https://user-images.githubusercontent.com/24864161/65587348-57f75100-dfa3-11e9-8487-f96d97c81840.png)


![Screenshot from 2019-09-25 14-39-28](https://user-images.githubusercontent.com/24864161/65587202-18c90000-dfa3-11e9-8537-cd1ce8fa9560.png)